devops/docker/README.md

```diff
@@ -0,0 +1,28 @@
+# docker image release workflow
+
+To login to dockerhub:
+```bash
+docker login
+```
+
+To build image:
+```bash
+docker build -t <org>/<image-name>:<version> .
+```
+
+To push a new tag:
+```bash
+docker push <org>/<image-name>:<version>
+```
```

Comment:
put here section as example:
https://github.com/checkstyle/contribution/pull/668

to be clear what is org name, image name , and how we do versioning.

Reply:
good idea, done